Day-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,121 @@
+.number {
+    font-size: 48px;
+    text-align: center;
+    padding-top: 20px;
+    
+}
+.number:hover{
+    cursor: pointer;
+    background-color: whitesmoke;
+}
+
+.action {
+    font-size: 48px;
+    text-align: center;
+    padding-top: 20px;
+    background-color: rgb(147, 28, 37);
+    color: white;	
+}
+
+.action:hover{
+    cursor: pointer;
+    background-color: lightgrey;
+    background-color: rgb(121, 22, 30);
+}
+
+.clear-button{
+    grid-column: 1 / 4;
+    grid-row: 2;
+    border: 1px solid grey;
+    font-weight: lighter;
+}
+
+.divide-button{
+    grid-column: 4;
+    grid-row: 2;
+    border: 1px solid grey;
+}
+
+.multiply-button{
+    grid-column: 4;
+    grid-row: 3;
+    border: 1px solid grey;
+}
+
+.minus-button{
+    grid-column: 4;
+    grid-row: 4;
+    border: 1px solid grey;
+}
+
+.add-button{
+    grid-column: 4;
+    grid-row: 5;
+    border: 1px solid grey;
+}
+
+.equals-button{
+    grid-column: 4;
+    grid-row: 6;
+    border: 1px solid grey;
+}
+
+.n0-button{
+    grid-column: 1 / 4;
+    grid-row: 6;
+    border: 1px solid grey;
+}
+
+.n1-button{
+    grid-column: 1;
+    grid-row: 5;
+    border: 1px solid grey;
+}
+
+.n2-button{
+    grid-column: 2;
+    grid-row: 5;
+    border: 1px solid grey;
+}
+
+.n3-button{
```

Comment:
I think there's a DRY-er way to use grid for this but I don't know enough about it.